src/PollbotAPI.js

```diff
@@ -7,7 +7,7 @@ import type {
   ReleaseInfo
 } from "./types";
 
-export const SERVER = "https://pollbot.dev.mozaws.net/v1";
+export const SERVER = "https://pollbot.services.mozilla.com/v1";
```

Comment:
Why isn't this configurable?

If it was:
```javascript
export const SERVER = process.env.REACT_APP_SERVER || "https://pollbot.services.mozilla.com/v1";
```
Then you'd be able to do:
```
$ REACT_APP_SERVER=https://pollbot.dev.mozaws.net/v1 yarn start
```
...if you'd need to test something new that's only on pollbot dev.